src/components/tooltip/__tests__/Tooltip.test.tsx

```diff
@@ -21,12 +21,13 @@ describe('components/tooltip/Tooltip', () => {
         sandbox.verifyAndRestore();
     });
 
-    describe('render()', () => {
+    describe.each([[true], [false]])('render() shouldStopEventPropagation=%o', shouldStopEventPropagation => {
```

Comment:
This seems a little excessive. The test should be targeted on things that the prop affects.

Reply:
will fix